File: lotto_api.py
import requests
from pprint import pprint as pp

url = 'https://www.dhlottery.co.kr/common.do?method=getLottoNumber&drwNo=874'
res = requests.get(url)

# json을 python dictionary로 파싱
data = res.json()

winner = []

# winner.append(data['drwtNo1'])
# winner.append(data['drwtNo2'])
# winner.append(data['drwtNo3'])
# winner.append(data['drwtNo4'])
# winner.append(data['drwtNo5'])
# winner.append(data['drwtNo6'])

for i in range(1, 7):
    winner.append(data[f'drwtNo{i}'])
    print(i)



print(winner)

File: lotto_api.js
const axios = require("axios");

// https://www.dhlottery.co.kr/common.do?method=getLottoNumber&drwNo=874

// let winner = [];
// const res = axios
//   .get("https://www.dhlottery.co.kr/common.do?method=getLottoNumber&drwNo=874")
//   .then(response => {
//     console.log(response.data);
//     let numData = "";
//     for (let i = 1; i < 7; i++) {
//       numData = `drwtNo${i}`;
//       console.log(numData);
//       winner.push(response.data.numData);
//       console.log(winner);
//     }
//   })
//   .catch(error => {
//     console.log(error);
//   });

url = "https://www.dhlottery.co.kr/common.do?method=getLottoNumber&drwNo=874";

axios.get(url).then(res => {
  let winner = [];
  for (let i = 1; i < 7; i++) {
    winner.push(res.data[`drwtNo${i}`]);
    console.log(winner);
  }
  console.log(winner);
});
